.drone.star

```diff
@@ -759,6 +779,7 @@ def localApiTestPipeline(ctx):
         "extraServerEnvironment": {},
         "storages": ["ocis"],
         "accounts_hash_difficulty": 4,
+        "emailNeeded": False,
```

Comment:
Maybe it is not necessary here
```suggestion
```

Reply:
It failed in CI with error `key "emailNeeded" not in dict`

Reply:
thought that it was added in different place. :+1: